docs/application/native/guides/text-input/autofill.md

```diff
@@ -345,7 +325,7 @@ To store entered data by the user in the autofill service:
         free(app_id);
     ```
 
-3. To store the data in an Autofill service using `autofill_commit()`, request with the `autofill_commit()` function:
+3. Request with the `autofill_commit()` function, to store the data in an Autofill service using `autofill_commit()`:
```

Comment:
To store the data in an Autofill service, request with the autofill_commit() function:"

Comment not fixed.

Reply:
In other place "~~~, to ~~~" pattens are used like:
"Create an Autofill data for each input fields, to save:" in line 295
"Create an Autofill view data, to include multiple Autofill items:" in line 309.

May I replace as 
"Request with the autofill_commit() function, to store the data in an Autofill service:"?

Reply:
@imfine98, sure you can change the sentence as suggested:
"Request with the autofill_commit() function, to store the data in an Autofill service:"?

Reply:
Fixed.